.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  pull_request:
```

Comment:
Is the trailing colon necessary?

Reply:
[Yes](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration)

> You must append a colon (:) to all events, including events without configuration.